toolchain/check/cpp/operators.cpp

```diff
@@ -190,10 +190,12 @@ auto LookupCppOperator(Context& context, SemIR::LocId loc_id, Operator op,
     return SemIR::ErrorInst::InstId;
   }
 
+  clang::SourceLocation loc = GetCppLocation(context, loc_id);
+  clang::OverloadCandidateSet::OperatorRewriteInfo operator_rewrite_info(
+      *op_kind, loc, true);
```

Comment:
Optional: Something like this might be clearer to someone unfamiliar with Clang:

```suggestion
      *op_kind, loc, /*AllowRewritten=*/true);
```

Reply:
Done.